Sources/Subprocess/Platforms/Subprocess+Darwin.swift

```diff
@@ -166,6 +166,9 @@ extension Configuration {
         error: Error,
         errorPipe: CreatedPipe
     ) throws -> Execution<Output, Error> {
+        #if !os(macOS)
```

Comment:
The motivation of this change makes sense, but this runtime error inside this function is very non obvious to me.